tests/testsuite/cargo_tree/deps.rs

```diff
@@ -1156,9 +1163,9 @@ fn format() {
             repository = "https://github.com/rust-lang/cargo"
 
             [dependencies]
-            dep = {version="1.0", optional=true}
+            dep = {version="=1.0"}
```

Comment:
This is removing coverage.

I suspect the test changes here should be purely additive rather than mutating existing stuff.